src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,79 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
 
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user.setId(1L);
+        user.setAge(21);
```

Comment:
```suggestion
        Storage.people.clear();
        user.setAge(21);
```

we dont need to set id here(it set bystorageDao)